tests/test_environment_anaysis.py

```diff
@@ -0,0 +1,149 @@
+import copy
+import datetime
+import os
+from unittest.mock import patch
+
+import ipywidgets as widgets
+import matplotlib as plt
+import pytest
+from IPython.display import HTML
+
+from rocketpy import EnvironmentAnalysis
+
+plt.rcParams.update({"figure.max_open_warning": 0})
+
+# Create a simple object of the Environment Analysis class
+@pytest.fixture
+def env():
+    """Create a simple object of the Environment Analysis class to be used in
+    the tests. This allows to avoid repeating the same code in all tests.
+
+    Returns
+    -------
+    EnvironmentAnalysis
+        A simple object of the Environment Analysis class
+    """
+    env = EnvironmentAnalysis(
+        start_date=datetime.datetime(2019, 10, 23),
+        end_date=datetime.datetime(2021, 10, 23),
+        latitude=39.3897,
+        longitude=-8.28896388889,
+        start_hour=6,
+        end_hour=18,
+        surfaceDataFile="./data/weather/EuroC_single_level_reanalysis_2002_2021.nc",
+        pressureLevelDataFile="./data/weather/EuroC_pressure_levels_reanalysis_2001-2021.nc",
+        timezone=None,
+        unit_system="metric",
+        forecast_date=None,
+        forecast_args=None,
+        maxExpectedAltitude=None,
+    )
+
+    return env
```

Comment:
Shouldn't this be in `conftest.py`?

Reply:
Yeah! I've just moved everything.

Btw the conftest.py may be starting to become confusing or difficult to read...